README.md

```diff
@@ -188,6 +188,29 @@ change the global `survey.PageSize`, or set the `PageSize` field on the prompt:
 prompt := &survey.MultiSelect{..., PageSize: 10}
 ```
 
+### Custom filtering in Select and MultiSelect
```

Comment:
Thanks for adding documentation!

Reply:
Since this is something that is shared across the select and multi select prompts, should we add something to the table of contents?